src/DataLayouts/DataLayouts.jl

```diff
@@ -574,6 +591,12 @@ end
 function Base.size(data::IJF{S, Nij}) where {S, Nij}
     return (Nij, Nij, 1, 1, 1)
 end
+function Base.fill!(data::IJF{S, Nij}, val) where {S, Nij}
+    for j in 1:Nij, i in 1:Nij
```

Comment:
```suggestion
    @inbounds for j in 1:Nij, i in 1:Nij
```
We should still add `@inbounds` here, right?